### PR TITLE
Making consistent footer layout

### LINKED ISSUE #113: Inconsistency in footer layout compare to other page footer layout on layer5
#### Description

On service-mesh-pattern website the footer layout is not consistent with other pages footer layout on layer5.
https://service-mesh-patterns.github.io/service-mesh-patterns/

Behavior
Screenshot 2023-07-19 201855

#### Expected Behavior

Screenshot 2023-07-19 201930

#### Environment:
- Meshery Version: 
- Kubernetes Version: 
- Host OS:
- Browser:

### PR DESCRIPTION
**Pull request by Cvr421**
**Description**
On service-mesh-pattern website the footer layout is not consistent with other pages footer layout on layer5.
https://service-mesh-patterns.github.io/service-mesh-patterns/
This PR fixes #113

**Notes for Reviewers**
Current Behavior
Screenshot 2023-07-19 201855

Expected Behavior
pattern

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
 Yes, I signed my commits.

### CODE REVIEW

**Comment by Cvr421** [comment]
Hii @leecalcote @Chadha93 @thisiskaransgit . I have made the changes into footer section please review and let me know your thought.

**Comment by l5io** [comment]
🚀 Preview for commit 3af3b605812af093f0fd5d1a2cbb578e927528dd at: https://64bbdecdb6caba69f8982ea1--service-mesh-patterns.netlify.app

File: site/src/components/Footer/footer.style.js [hunk 1]
```diff
@@ -1,235 +1,328 @@
...
-  width: 100%;
-  height: auto;
-  background-color: black;
-  align-items: center;
-  padding-top: 3.5rem;
-  gap: 2.5rem;
-`;
+const FooterWrapper = styled.section`
+	padding: 2.5rem 3rem 1.5rem;
+	background: black;
```
Comment (Chadha93) [inline site/src/components/Footer/footer.style.js:5]: @Cvr421 I see you are using theme ref for colors on line 49. Good. Let's be consistent and use the theme here too.

File: site/src/components/Footer/footer.style.js [hunk 2]
```diff
@@ -1,235 +1,328 @@
...
-    p {
-      font-size: 1.2rem;
-    }
-    img {
-      width: 22rem;
-    }
-  }
-`;
+	.desc-info {
+		color: #fff;
```
Comment (Chadha93) [inline site/src/components/Footer/footer.style.js:27]: here too.

File: site/src/components/Footer/footer.style.js [hunk 3]
```diff
@@ -1,235 +1,328 @@
...
-    }
-  }
-  @media (max-width: 480px) {
-    align-items: center;
-  }
-`;
+		.section-title{
+			font-size: 1.25rem;
+			margin-bottom: 1rem;
+			color: rgba(255, 255, 255, 0.8);
```
Comment (Chadha93) [inline site/src/components/Footer/footer.style.js:43]: here too.

File: site/src/components/Footer/footer.style.js [hunk 4]
```diff
@@ -1,235 +1,328 @@
...
-  @media (min-width: 481px) and (max-width: 767px) {
-    text-align: center;
-    padding-top: 2rem;
-  }
-`;
+			.title-link {
+				transition: 0.2s all;
+				color: rgba(255, 255, 255, 0.8);
+				&:hover {
+					color: ${(props) => props.theme.secondaryColor};
```
Comment (Chadha93) [inline site/src/components/Footer/footer.style.js:49]: Good.

File: site/src/components/Footer/footer.style.js [hunk 5]
```diff
@@ -1,235 +1,328 @@
...
+				color: #CCCCCC;
+				transition: 0.2s all;
+
+				&:hover{
+					color: ${(props) => props.theme.secondaryColor};
+				}
+			}
+
+			.new-label {
+				background-color: rgba(255, 255, 255, 0.25);
```
Comment (Chadha93) [inline site/src/components/Footer/footer.style.js:70]: theme please.

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@Cvr421 Thanks for the PR, there are multiple lines where you missed using them ref for colors.

**Comment by Cvr421** [comment]
> @Cvr421 Thanks for the PR, there are multiple lines where you missed using them ref for colors.

Ok @Chadha93 . I'll make the changes that you recommended . And then I'll let you know. 
Thanks

File: site/src/components/Footer/footer.style.js (hunk 5 above)
Comment (Cvr421) [reply site/src/components/Footer/footer.style.js:70]: Hii @Chadha93 actually this color { background-color: rgba(255, 255, 255, 0.25); } is not defined in theme reference 
so should defined these color in theme ref  and then make a change or we can go with this  background-color: rgba(255, 255, 255, 0.25.

**Comment by leecalcote** [comment]
Thanks, but no thanks, @Cvr421. I previously offered explanation as to why and also offered a list of higher priority items to dig into. Please do. Let's not have @Chadha93 spend more of his time here, though.

**Comment by Cvr421** [comment]
> Thanks, but no thanks, @Cvr421. I previously offered explanation as to why and also offered a list of higher priority items to dig into. Please do. Let's not have @Chadha93 spend more of his time here, though.

Sorry @leecalcote but i am focusing on both the things. Actually this PR is almost completed so i thought to open this PR and let's finish it.

**Comment by leecalcote** [comment]
Thanks, I really do appreciate this, however, it's not just your time and focus that we need to account for but my time in offering an explanation for the third time and @Chadha93's time in reviewing, others in qualifying the issue and then when the bugs roll in from the inevitable bug / non-responsive issue / wrong image size / ...... /.... /.... then it's just more churn on something that suffices for now while we prioritize focus on other work.

**Comment by Cvr421** [comment]
> Thanks, I really do appreciate this, however, it's not just your time and focus that we need to account for but my time in offering an explanation for the third time and @Chadha93's time in reviewing, others in qualifying the issue and then when the bugs roll in from the inevitable bug / non-responsive issue / wrong image size / ...... /.... /.... then it's just more churn on something that suffices for now while we prioritize focus on other work.

ok @leecalcote I completely  understand that Sorry once again .

**Pull request closed without merging**